app/question.js

```diff
@@ -100,20 +120,18 @@ define(['jquery', 'app/messages', 'app/config', 'app/string'], function ($, mess
       var difference = levenstein(input,answer);
 
       if (difference == 0) {
-        messages.show( "Well done!", "success", config.constant("FEEDBACK_DELAY") );
-        nextQuestion();
+        handleScoreIncrease();
+        messages.show( "Well done!", "success", config.constant("FEEDBACK_DELAY_CORRECT") );
       } else if (isWithinMarginOfError(answer, difference)) {
-        messages.show( "Almost there! Your answer: " + input + " - Expected answer: " + answer + " (" + difference + " letter" + string.pluralIfAppropriate(difference) + " difference)", "warning", config.constant("FEEDBACK_DELAY") );
-        currentItemIndex = (currentItemIndex + 1) % items.length;
+        messages.show( "Almost there! Expected answer: <b>" + answer + "</b> (" + difference + " letter" + string.pluralIfAppropriate(difference) + " difference)", "warning", config.constant("FEEDBACK_DELAY_INCORRECT") );
```

Comment:
Use a method to build this string, or move the whole thing to the view and update spans in the `<p>` element.